apps/etterlatte-saksbehandling-ui/server/src/routers/githubRouter.ts

```diff
@@ -0,0 +1,37 @@
+import express, { Request, Response } from 'express'
```

Comment:
Trenger vi å ha noe i frackend i det hele tatt egentlig, hvorfor ikke bare hente jevnlig fra frontend og vise frem det?

Reply:
Mindre spamming av GH API-et når vi kan cache responsen på tvers av brukerne. Tror ikke de har noen begrensninger på antall kall, men synes det er greit å redusere kall mot tredjepart hvis vi kan det. 

Og vi får da en slags "backup" (dog kortvarig) for de tilfellene hvor GH har nedetid.